src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -409,6 +409,12 @@ private FulfilmentRequest createFulfilmentRequestPayload(
     Product product = findProduct(fulfilmentCode, deliveryChannel, region);
 
     if (deliveryChannel.equals(Product.DeliveryChannel.POST)) {
```

Comment:
I appreciate that you haven't written this line but it could be improved by treating it as enum rather than an object. ie:
`    if (deliveryChannel == Product.DeliveryChannel.POST) {`
Continuous improvement :)

Reply:
that's fine, I'm all for continuous improvement : )